src/SCRIPTS/BF/X7/x7pre.lua

```diff
@@ -1,5 +1,6 @@
-PageFiles = 
+PageFiles =
 {
+    "vtx.lua",
```

Comment:
This ordering change is entirely subjective, and will probably be disliked by just as many users as it will be liked by, plus it will be disliked by all users that do not like a change from what they are used to, so I don't think this should be done without a good reason.

Reply:
OMG did I do this?
I only wanted to change these on my own github 0crap account and not at all push into the official master. Only do the minor textual improvement back into the master. Can you tell me what I did wrong?